packages/aws-cdk/lib/init-templates/v2/app/csharp/src/%name.PascalCased%/%name.PascalCased%.template.csproj

```diff
@@ -10,7 +10,6 @@
   <ItemGroup>
     <!-- CDK Construct Library dependencies -->
     <PackageReference Include="Amazon.CDK.Lib" Version="%cdk-version%" />
-    <PackageReference Include="Constructs" Version="%constructs-version%" />
```

Comment:
From what I understand, peer deps is a concept only in npm. 
In other languages, constructs is already declared as a dependency of the CDK, and will be installed as any dependency of the cdk will be.

Reply:
Yeah it will be pulled as a transitive dep so it will technically work, but if your code directly uses some type from a library it's a bad practice to rely on a transitive dependency to pull that library. You should declare it as a direct dependency as well.

Reply:
Fair point

Reply:
But then also... it doesn't need to be the same version that CDK uses, right? It can be any version that's compatible with that one.

Can we just hardcode `constructs >= 10, < 11` ?

Reply:
Yeah. Should be safe to hard code